worklist/wikipedia_utils.py

```diff
@@ -4,6 +4,9 @@
 
 # Function that uses MediaWiki API that maps page titles to IDs
 def convert_article_titles_into_ids(titles):
+    if len(titles) == 0:
```

Comment:
Just realized the PetScan result should contain the page ids: https://petscan.wmflabs.org/?format=json&psid=3316252

Maybe we don't actually need this separate call to the API?

Reply:
This call is needed because we want to store the title as that will be displayed on the front-end.

Reply:
We're also getting the title back from PetScan, no?

Reply:
Discussed live, we need to do this for articles added manually.